preprocessy/outliers/_handleoutlier.py

```diff
@@ -64,7 +54,7 @@ def __return_quartiles(self, col):
         q3 = round(q3)
         self.quartiles[col] = [q1, q3]
 
-    def handle_outliers(self):
+    def handle_outliers(self, params):
 
         # parameters till now: train_df, cols, removeoutliers, replace
         train_df = self.train_df
```

Comment:
`self.train_df` is `None` here. You have to first initialise `self.train_df = params["train_df"]` along with other initialisations.

Reply:
Will get it done.